src/site/content/en/blog/hands-on-portals/index.md

```diff
@@ -132,19 +132,19 @@ If you want to quickly experience what Portals feel like, try using [uskay-porta
 ## Check out the spec
 We are actively discussing [the spec](https://wicg.github.io/portals/) in the Web Incubation Community Group (WICG). To quickly get up to speed, take a look at [the explainer](https://github.com/WICG/portals/blob/master/explainer.md). These are the three important features to familiarize yourself with:
 
- - [The portal element:](https://wicg.github.io/portals/#the-portal-element) The HTML element itself. The API is very simple. It consists of the `src` attribute, the `activate` function and an interface for messaging (`postMessage`). `activate` takes an optional argument to pass data to the portal upon activation.
- - [The portal host interface:](https://wicg.github.io/portals/#the-portalhost-interface) Adds a `portalHost` object to the `window` object. This lets you check if the page is embedded as a portal element. It also provides an interface for messaging (`postMessage`) back to the host.
- - [The PortalActivateEvent interface:](https://wicg.github.io/portals/#the-portalactivateevent-interface) An event that fires when the portal is activated. There is a neat function called `adoptPredecessor` which you can use to retrieve the previous page as a `<portal>` element. This allows you to create seamless navigations and composed experiences between two pages.
+ - [The `<portal>` element:](https://wicg.github.io/portals/#the-portal-element) The HTML element itself. The API is very simple. It consists of the `src` attribute, the `activate` function and an interface for messaging (`postMessage`). `activate` takes an optional argument to pass data to the `<portal>` upon activation.
+ - [The `<portal>` host interface:](https://wicg.github.io/portals/#the-portalhost-interface) Adds a `portalHost` object to the `window` object. This lets you check if the page is embedded as a `<portal>` element. It also provides an interface for messaging (`postMessage`) back to the host.
```

Comment:
```suggestion
 - [The `portal` host interface:](https://wicg.github.io/portals/#the-portalhost-interface) Adds a `portalHost` object to the `window` object. This lets you check if the page is embedded as a `<portal>` element. It also provides an interface for messaging (`postMessage`) back to the host.
```

I was wondering if it might make sense to only use the angle brackets when referring to the HTML element itself. In this case the portal host interface is referring to a DOM object associated with the portal.

Reply:
Yep. I just missed this one.

Reply:
Done.